packages/contracts/contracts/registry/PluginRepoRegistry.sol

```diff
@@ -56,6 +63,37 @@ contract PluginRepoRegistry is InterfaceBasedRegistry {
         emit PluginRepoRegistered(name, pluginRepo);
     }
 
+    /// @notice Checks if the name is either 0-9, a-z or a dash (-).
+    /// @param _name The name of the plugin.
+    /// @return `true` if the name is valid or `false` if at least one char is invalid.
+    /// @dev Aborts on the first invalid char found.
+    function _checkNameValidity(string calldata _name) internal pure returns (bool) {
```

Comment:
Easier to have one file level function instead?  Looks good to me anyway

Reply:
done